deploy/templates/_helpers.tpl

```diff
@@ -39,7 +39,8 @@ Create chart name and version as used by the chart label.
 Define the image for a container.
 */}}
 {{- define "amrc-connectivity-stack.image-name" -}}
-{{- .image.registry }}/{{ .image.repository }}:{{ .image.tag }}
+{{- $defaultTag := default .Release.Version .image.tag }}
```

Comment:
I think this is still scoped within the `.Values.*` passed in as context, so `.Release.Version` won't work. We need `$.Release.Version`.

I think it's also worth making this `coalesce .image.tag $.acs.defaultTag $.Release.Version` or something to allow a global override of the image tag. In particular I think a Flux HelmRelease pulling directly from a GitRepository results in a weird `.Release.Version` which won't work correctly.